javaparser-core-testing/src/test/java/com/github/javaparser/ast/validator/Java14ValidatorTest.java

```diff
@@ -72,7 +89,7 @@ class RecordDeclarationForbidden {
             void recordDeclaration() {
                 String s = "record X() { }";
                 ParseResult<CompilationUnit> result = javaParser.parse(COMPILATION_UNIT, provider(s));
-                TestUtils.assertProblems(result, "(line 1,col 1) Record Declarations are not supported. Pay attention that this feature is supported starting from 'JAVA_14' language level. If you need that feature the language level must be configured in the configuration before parsing the source files.");
```

Comment:
This was Intellij's auto formatting. I could change it back, but the change is more consistent with how other tests do it.